lib/kramdown/converter/pdf.rb

```diff
@@ -149,16 +149,11 @@ def render_standalone_image(el, opts)
         elsif img.attr['src'] !~ /\.jpe?g$|\.png$/
           warning("Cannot render images other than JPEG or PNG, " \
                   "got #{img.attr['src']}#{line ? " on line #{line}" : ''}")
-          return nil
         end
 
-        img_dirs = (@options[:image_directories] || ['.']).dup
-        begin
-          img_path = File.join(img_dirs.shift, img.attr['src'])
-          image_obj, image_info = @pdf.build_image_object(open(img_path))
-        rescue StandardError
-          img_dirs.empty? ? raise : retry
-        end
+        img_dirs = [".", ""] + @options.fetch(:image_directories, [])
```

Comment:
Please don't change the order of things, i.e. the `image_directories` should come first.